.github/workflows/docs_gh_pages.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy Docs to GitHub Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["master"]
```

Comment:
Seems good to me! I think only thing is let's not run this in master branch pushes, since master branch changes won't impact the docs (unless this file is changed).